### DISCUSSION TITLE
GItHub Education approved but not available

### DISCUSSION THREAD

**Discussion by MarkovicLuka95** [community/community#131918] [GitHub Education]
### Hi everyone,

I applied for the GitHub student pack on 16th Jan, my application is approved, but it is still unavailable.

**Comment by paiineli** [comment]
Hi,

Here are a few steps you can take:

Check Your Email: Make sure you’ve received any confirmation email regarding your approval. Sometimes, there might be additional steps needed.

Clear Cache and Cookies: Try clearing your browser's cache and cookies, then log in to GitHub again to see if the pack becomes available.

Verify Account Details: Ensure that your educational email is properly verified and linked to your GitHub account.

Contact GitHub Support: If the issue persists, I recommend reaching out to GitHub Support directly for assistance. They can provide specific insights into your account status.

I hope this helps, and that you gain access soon!

**Comment by samus-aran** [comment]
Hi @MarkovicLuka95 👋 

We've passed this along to our internal team. If we have any updates we'll let you know in this thread. 👍

**Reply by MarkovicLuka95** [reply]
Thank you!

**Reply by samus-aran** [reply]
Hi @MarkovicLuka95 

That should be you approved and have received your benefits. Enjoy!

**Reply by ramonlejandro0226** [reply]
hey me too i have received my aproved  i didn`t see mi benefits
